ms_auth/api/api_user.py

```diff
@@ -56,7 +57,7 @@ def register(request):
 			json_response = json.dumps(response)
 			response = HttpResponse(json_response, content_type='application/json', status=200)
 			response.set_cookie('jwt_token', jwt_token, httponly=True)
-			two_factor_views.send_welcome_email(user_id, user_views.getValue(user_id, 'email'))
+			mail_views.send_welcome_email(username, user_views.getValue(user_id, 'email'))
```

Comment:
i think we should not send a welcome email, we use mail for authentification not as a part of our application